apps/web/e2e/api/tags.spec.ts

```diff
@@ -0,0 +1,384 @@
+import { test, expect } from "@playwright/test";
+import type { APIRequestContext } from "@playwright/test";
+
+test.describe("Tags API", () => {
```

Comment:
simplify and reduce the number of test, create onle ONE test.describe, the other should be just tests